packages/http/README.md

```diff
@@ -180,24 +180,9 @@ The actual interface looks like this (but rather than explain each property we'r
 ```ts
 export interface IHttpConfig extends IPrismConfig {
   mock: false | IHttpOperationConfig;
-
-  validate?: {
-    request?:
-      | boolean
-      | {
-          hijack?: boolean;
-          headers?: boolean;
-          query?: boolean;
-          body?: boolean;
-        };
-
-    response?:
-      | boolean
-      | {
-          headers?: boolean;
-          body?: boolean;
-        };
-  };
+  cors: boolean;
```

Comment:
Curious how did `cors` got into this PR?

Reply:
I forgot about it in the previous PR that was making the cors feature happen.